src/pages/video-listing/video-listing.js

```diff
@@ -0,0 +1,95 @@
+import axios from 'axios'
+import './video-listing.css'
+import { useEffect } from 'react'
+import {Navbar} from '../../components/navbar/navbar'
+import { Sidebar } from '../../components/sidebar/sidebar'
+import { useVideo } from '../../context/video-context'
+export const VideoListing=()=>{
+
+    const {videoItem,setVideoItem,watchLater,setWatchLater}=useVideo()
+   
+
+    useEffect(()=>{
+        videos()
+    })
+
+    const videos= async ()=>{
+        const response=await axios.get('/api/videos')
+        setVideoItem(response.data.videos);
+    }
+
+    const watchLaterHandler= async (video)=>{
```

Comment:
try to put into another file so code looks clean and easy to debug

Reply:
ok will do it